modules/general/acr/README.md

```diff
@@ -0,0 +1,39 @@
+# container-registry
+
+Azure Container Registry
+
+## Description
+
+{{ Add detailed description for the module. }}
```

Comment:
Is this now auto-generated as placeholder text?

Reply:
There are bunch of those across the modules resulting from the updates to the `brm` tooling - I was thinking we can address those in a follow-up PR, which I've now added a task for: #14